k8s/aat/common-overlay/bsp/kustomization.yaml

```diff
@@ -2,4 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: bsp
 bases:
-- ../../../namespaces/bsp
+  - ../../../namespaces/bsp
+patchesStrategicMerge:
+  - ../../../namespaces/bsp/bulk-scan-payment-processor/aat.yaml
```

Comment:
couldn't see this file

Reply:
good catch. i've used as `git mv` on most files. this one probably was manually created and i missed it ;)

Reply:
Anyway, PR checks failed :)